gitlab-pipeline/stage/trigger-images.yml

```diff
@@ -18,6 +14,10 @@ trigger:mender-convert:
     # so it cannot be passed downstream to mender-convert.
     # It is not installed in our pre-converted images, so it doesn't really matter
     #MENDER_ADDON_CONFIGURE_VERSION
+
+    # Mode: "build and publish" or "build and test"
+    TEST_MENDER_CONVERT: $TEST_MENDER_CONVERT
+    PUBLISH_MENDER_CONVERT_AUTOMATIC: $PUBLISH_RELEASE_AUTOMATIC
```

Comment:
Will you remove this as we talked about, or is it just that it takes its value from `PUBLISH_RELEASE_AUTOMATIC`?

Same with dist-packages.

Reply:
This just takes the mender-qa variable `PUBLISH_RELEASE_AUTOMATIC` and translates to the mender-convert one. I could change in mender-convert/mender-dist-packages to use the same name, but I think I prefer it as it is.